test/integration/prerender/test/index.test.js

```diff
@@ -264,6 +264,16 @@ const runTests = (dev = false) => {
     expect(JSON.parse(params)).toEqual({})
   })
 
+  it('should not supply query values to params in /_next/data request', async () => {
+    const data = JSON.parse(
+      await renderViaHTTP(
+        appPort,
+        `/_next/data/${buildId}/something.json?hello=world`
+      )
+    )
+    expect(data.pageProps.params).toEqual({})
```

Comment:
The `query` should be empty too, right? Not just params.

Reply:
The `query` isn't passed in `getStaticProps`. We check it's not supplied during rendering in `useRouter` in the above test